spec/index.bs

```diff
@@ -769,16 +771,42 @@ GPUBuffer includes GPUObjectBase;
     : <dfn>\[[state]]</dfn> of type [=buffer state=].
     ::
         The current state of the {{GPUBuffer}}.
+
+    : <dfn>\[[mapping]]</dfn> of type {{ArrayBuffer}}.
```

Comment:
should we merge it with the other one, saying that this attribute is either:
  - null
  - mapping
  - promise of mapping